ChangeLog

```diff
@@ -37,6 +37,10 @@ What's New in astroid 3.2.3?
 ============================
 Release date: 2024-07-11
 
+
+* Implement inference for JoinedStr and FormattedValue
```

Comment:
Please move this to the 3.3.0 milestone above.

Reply:
Done